app/services/assessors/property_assessor.rb

```diff
@@ -1,22 +1,78 @@
 module Assessors
   class PropertyAssessor
-    Result = Struct.new(:transaction_allowance,
-                        :net_value,
-                        :net_equity,
-                        keyword_init: true)
+    Result = Data.define(
+      :transaction_allowance,
+      :net_value,
+      :net_equity,
+      :main_home_equity_disregard,
+      :assessed_equity,
+      :smod_allowance,
+    ) do
+      def self.blank
+        new(0, 0, 0, 0, 0, 0)
```

Comment:
I wrote this is in a hurry - should we use keywords? [It seems harmless at the moment because the code is right next to the definition)